core/src/main/java/org/infinispan/stats/impl/CacheContainerStatsImpl.java

```diff
@@ -64,7 +64,11 @@ public void resetStatistics() {
       if (getStatisticsEnabled()) {
          for (String cn : cm.getCacheNames()) {
             if (cm.cacheExists(cn)) {
-               cm.getCache(cn).getAdvancedCache().getStats().reset();
+               try {
+                  cm.getCache(cn).getAdvancedCache().getStats().reset();
+               } catch (SecurityException e) {
+                  // no-op, skipping this cache
```

Comment:
sorry for poor description, I've updated [ISPN-6261](https://issues.jboss.org/browse/ISPN-6261) with steps how to reproduce this issue. Let me know if it's not sufficient or anything is unclear. Basically, I don't do anything special, just start CLI, navigate to cache container and run `ls` there